packages/@mantine/core/src/components/Tooltip/use-tooltip.ts

```diff
@@ -90,7 +89,6 @@ export function useTooltip(settings: UseTooltip) {
     useRole(context, { role: 'tooltip' }),
     // cannot be used with controlled tooltip, page jumps
     useDismiss(context, { enabled: typeof settings.opened === 'undefined' }),
-    useDelayGroup(context, { id: uid }),
```

Comment:
Removing this will break functionality of Tooltip. It should not be removed.

Reply:
It has no effect. group delay is already defined here: 

`const { delay: groupDelay, currentId, setCurrentId } = useDelayGroupContext();`

and applied to the tooltip (/ group) here : 

`delay: withinGroup ? groupDelay : { open: settings.openDelay, close: settings.closeDelay },`

I tested this change on Storybook, works without any issues. 

The reason I removed this line is the TS error I had shared in the screenshot in our conversation on the support channel.

Reply:
I cannot merge PR with this change, please remove it and keep PR specific to storybook